### PR TITLE
Breathing - Add BVM and O2Tank to Supply Crates

### PR DESCRIPTION
**Pull request by Apricot-ale**
**When merged this pull request will:**
- _Add BVM and oxygenTank to KAM General Items / ACE Advanced Medical Supply Crate_
- _Add missing scopeCurator & scopeArsenal config to BVM and oxygenTank_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by Apricot-ale** [comment]
This PR maybe solve this issue https://github.com/KAT-Advanced-Medical/KAM/issues/423

NOPE. it's not solved

File: addons/breathing/CfgVehicles.hpp [hunk 1]
```diff
@@ -55,6 +59,12 @@ class CfgVehicles {
             MACRO_ADDITEM(KAT_Empty_bloodIV_500,7);
             MACRO_ADDITEM(kat_AED,2);
             MACRO_ADDITEM(kat_X_AED,2);
+            MACRO_ADDITEM(kat_BVM,5);
+            MACRO_ADDITEM(kat_pocketBVM,5);
+            MACRO_ADDITEM(kat_oxygenTank_150_Item,2);
+            MACRO_ADDITEM(kat_oxygenTank_300_Item,2);
+            MACRO_ADDITEM(kat_oxygenTank_150_Empty,2);
```
Comment (MiszczuZPolski) [inline addons/breathing/CfgVehicles.hpp]: why adding empty tanks tho?

Comment (Apricot-ale) [reply addons/breathing/CfgVehicles.hpp]: I added it with a macro along with BVM, so I'll revert it.

**Comment by Apricot-ale** [comment]
> Looks not solved yet with latest version Antistasi... :/

Maybe this requires action on the part of Antistasi!

**Pull request merged**